webview-src/index.ts

```diff
@@ -79,42 +83,58 @@ export default class Database {
    * ```
    */
   async execute(query: string, bindValues?: unknown[]): Promise<QueryResult> {
-    return await invoke<[number, number]>('plugin:sql|execute', {
-      db: this.path,
-      query,
-      values: bindValues ?? []
-    }).then(([rowsAffected, lastInsertId]) => ({ rowsAffected, lastInsertId }))
+    const [rowsAffected, lastInsertId] = await invoke<[number, number]>(
+      'plugin:sql|execute',
+      {
+        db: this.path,
+        query,
+        values: bindValues ?? []
+      }
+    )
+
+    return {
+      lastInsertId,
+      rowsAffected
+    }
   }
 
   /**
    * **select**
    *
    * Passes in a SELECT query to the database for execution.
    *
+   * @example
    * ```ts
    * const result = await db.select(
    *    "SELECT * from todos WHERE id = $1", id
    * );
    * ```
    */
   async select<T>(query: string, bindValues?: unknown[]): Promise<T> {
-    return await invoke('plugin:sql|select', {
+    const result = await invoke<T>('plugin:sql|select', {
       db: this.path,
       query,
       values: bindValues ?? []
     })
+
+    return result
```

Comment:
What's the value here? Just making it consistent with the 2 changes above?

Reply:
Do you mean by declaring a new variable and returning that instead of the function invocation itself?
It just makes sense to keep the style the same as the above functions before this.
The most crucial goal here is to make it more readable.

Reply:
not feeling it but who am i to judge